src/main/java/org/dinosaur/foodbowl/domain/comment/application/CommentService.java

```diff
@@ -0,0 +1,49 @@
+package org.dinosaur.foodbowl.domain.comment.application;
+
+import static org.dinosaur.foodbowl.global.exception.ErrorStatus.MEMBER_NOT_FOUND;
+import static org.dinosaur.foodbowl.global.exception.ErrorStatus.POST_NOT_FOUND;
+
+import lombok.RequiredArgsConstructor;
+import org.dinosaur.foodbowl.domain.comment.dto.CommentRequest;
+import org.dinosaur.foodbowl.domain.comment.dto.CommentResponse;
+import org.dinosaur.foodbowl.domain.comment.entity.Comment;
+import org.dinosaur.foodbowl.domain.comment.repository.CommentRepository;
+import org.dinosaur.foodbowl.domain.member.entity.Member;
+import org.dinosaur.foodbowl.domain.member.repository.MemberRepository;
+import org.dinosaur.foodbowl.domain.post.entity.Post;
+import org.dinosaur.foodbowl.domain.post.repository.PostRepository;
+import org.dinosaur.foodbowl.global.exception.FoodbowlException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+@Service
+public class CommentService {
+
+    private final PostRepository postRepository;
+    private final MemberRepository memberRepository;
+    private final CommentRepository commentRepository;
+
+    @Transactional
+    public CommentResponse save(Long memberId, CommentRequest commentRequest) {
+        Long postId = commentRequest.getPostId();
+        Post post = postRepository.findById(postId)
+                .orElseThrow(() -> new FoodbowlException(POST_NOT_FOUND));
```

Comment:
```suggestion
        Post post = postRepository.findById(commentRequest.getPostId())
                .orElseThrow(() -> new FoodbowlException(POST_NOT_FOUND));
```
`request`에서 바로 사용하는건 어떤가요?!

Reply:
저는 개인적으로 꺼내서 쓰는게 가독성이 좋아서 해당 방법을 선호하긴 하는데 
여기서는 굳이 안꺼내도 get필드로 확인할 수 있겠네요 ~